.github/workflows/zizmor.yml

```
@@ -7,17 +7,16 @@ on:
       - main
       - '22.0'
     paths:
+    paths: &paths
```

actionlint (1.7.12)

[error] key "paths" is duplicated in "push" section. previously defined at line:9,col:5

(syntax-check)